geode-core/src/main/java/org/apache/geode/cache/configuration/JndiBindingsType.java

```diff
@@ -627,104 +626,90 @@ public String getId() {
      *
      */
     @XmlAccessorType(XmlAccessType.FIELD)
-    @XmlType(name = "",
-        propOrder = {"configPropertyName", "configPropertyType", "configPropertyValue"})
+    @XmlType(name = "", propOrder = {"name", "type", "value"})
     public static class ConfigProperty implements CacheElement {
 
+      private static final long serialVersionUID = 1L;
       @XmlElement(name = "config-property-name", namespace = "http://geode.apache.org/schema/cache",
           required = true)
-      protected String configPropertyName;
+      protected String name;
```

Comment:
I think we changed the name to be "configPropertyName" for some json serialization which requires this name. Do all tests pass?